web/src/components/apps/AppConfig.jsx

```diff
@@ -513,22 +516,77 @@ class AppConfig extends Component {
             })}
           </div>
           <div className="ConfigArea--wrapper">
-            {this.renderConfigInfo(app)}
-            <div className={classNames("ConfigOuterWrapper u-paddingTop--30", { "u-marginTop--20": fromLicenseFlow })}>
-              <div className="ConfigInnerWrapper">
-                <AppConfigRenderer groups={configGroups} getData={this.handleConfigChange} readonly={this.isConfigReadOnly(app)} configSequence={match.params.sequence} appSlug={app.slug} />
-              </div>
-            </div>
-            {savingConfig ?
-              <div className="u-paddingBottom--30">
-                <Loader size="30" />
-              </div>
-              :
-              <div className="ConfigError--wrapper flex-column u-paddingBottom--30 alignItems--flexStart">
-                {configError && <span className="u-textColor--error u-marginBottom--20 u-fontWeight--bold">{configError}</span>}
-                <button className="btn primary blue" disabled={!changed && !fromLicenseFlow || this.isConfigReadOnly(app)} onClick={this.handleSave}>{fromLicenseFlow ? "Continue" : "Save config"}</button>
-              </div>
-            }
+            <IsHelmManaged>
```

Comment:
This is a WIP- will refactor `AppConfig` into a functional component so we don't have to use render props.